app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -14,9 +14,9 @@
             android:icon="@drawable/ic_menu_gallery"
             android:title="Artists" />
         <item
-            android:id="@+id/nav_genre"
+            android:id="@+id/nav_albums"
             android:icon="@drawable/ic_menu_slideshow"
-            android:title="Genre" />
+            android:title="Albums" />
```

Comment:
move this to `strings.xml` and reference using `@string/...`, same with other uses elsewhere if you are using strings that aren't in your string resources file 👍